source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -123,6 +127,17 @@ std::string AuthenticatorImpl::name() const {
   return "_UNKNOWN_";
 }
 
+std::string AuthenticatorImpl::convertDoubleToString(double double_value) {
+  double int_part;
```

Comment:
Please add comment for this function as
// If the number is unsigned 64 bit integer,  convert to string as integer,
// otherwise, convert to string as double.